sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2514,6 +2515,11 @@ def _get_pipeline_details(unused_elm):
                 **self._kwargs))
         | _PassThroughThenCleanupTempDatasets(project_to_cleanup_pcoll))
 
+  def get_pcoll_from_schema(table_schema):
+    pcoll_val = apache_beam.io.gcp.bigquery_schema_tools.\
+        produce_pcoll_with_schema(table_schema)
+    return beam.Map(lambda values: pcoll_val(**values))
```

Comment:
The integration test should build a pipeline using this method and run it, similar to this test: https://github.com/apache/beam/blob/8c4a056a63d92776ae9d6be726b37d789486afbd/sdks/python/apache_beam/io/gcp/bigquery_read_it_test.py#L350-L356

Does that make sense?

Reply:
You might need to a `beam.Map().with_output_types(pcoll_val)`, that's how Beam knows the element type